src/main/java/com/hollingsworth/arsnouveau/common/block/SourceBerryBush.java

```diff
@@ -86,7 +86,7 @@ public void entityInside(BlockState state, Level worldIn, BlockPos pos, Entity e
                 double d0 = Math.abs(entityIn.getX() - entityIn.xOld);
                 double d1 = Math.abs(entityIn.getZ() - entityIn.zOld);
                 if (d0 >= (double)0.003F || d1 >= (double)0.003F) {
-                    entityIn.hurt(DamageSource.SWEET_BERRY_BUSH, 1.0F);
+                    entityIn.hurt(new DamageSource("sourceberrybush"), 1.0F);
```

Comment:
Why was this changed?

Reply:
It's only used here so I decided to create a throwaway instance. And no, it shouldn't be registered anywhere.